index.js

```diff
@@ -6,16 +6,16 @@
 // So if we want our explicit definitions to extend the default, we do this:
 const strictValueDefaults = ['inherit', 'initial', 'unset'];
 
-// See https://stylelint.io/user-guide/example-config/
+// See https://stylelint.io/user-guide/configure
 module.exports = {
-    "syntax": "scss",
+    "customSyntax": "postcss-scss",
```

Comment:
on why this has been changed to `postcss-scss` is because when I tried the upgraded `stylelint-config-foxsports` in infinity I would get the following error when running stylelint command:


```
➜  infinity git:(develop) ✗ npm run lint:style

> @kayo/infinity@2.26.6 lint:style /Users/nesa.mouzehkesh/infinity
> stylelint "src/**/*.js" ".storybook/**/*.js" --color && echo '✓ lint:style'

Error: Cannot resolve custom syntax module "scss". Check that module "scss" is available and spelled correctly.
    at getCustomSyntax (/Users/nesa.mouzehkesh/infinity/node_modules/stylelint/lib/getPostcssResult.js:99:11)
    at getPostcssResult (/Users/nesa.mouzehkesh/infinity/node_modules/stylelint/lib/getPostcssResult.js:41:5)
    at lintSource (/Users/nesa.mouzehkesh/infinity/node_modules/stylelint/lib/lintSource.js:76:20)
    at async /Users/nesa.mouzehkesh/infinity/node_modules/stylelint/lib/standalone.js:218:27
    at async Promise.all (index 0)
    at async standalone (/Users/nesa.mouzehkesh/infinity/node_modules/stylelint/lib/standalone.js:257:22)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @kayo/infinity@2.26.6 lint:style: `stylelint "src/**/*.js" ".storybook/**/*.js" --color && echo '✓ lint:style'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @kayo/infinity@2.26.6 lint:style script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nesa.mouzehkesh/.npm/_logs/2021-11-01T23_10_07_342Z-debug.log
```

so then when I searched into the description of `customSyntax` in stylelint docs in [here](https://stylelint.io/user-guide/usage/options#customsyntax) and also a few raised issues in github such as [this](https://github.com/stylelint-scss/stylelint-config-standard-scss/issues/2) I installed `postcss` and `postcss-scss` and passed `postcss-scss` to the `customSyntax` and the above error went away. And now when I run `npm run lint:style` I get the linting errors based on my changes which is cool but right at the end of it I get this error:

```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @kayo/infinity@2.26.6 lint:style: `stylelint "src/**/*.js" ".storybook/**/*.js" --color && echo '✓ lint:style'`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @kayo/infinity@2.26.6 lint:style script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nesa.mouzehkesh/.npm/_logs/2021-11-01T23_13_46_480Z-debug.log
```

I am currently looking into that error...

Reply:
ummm, still experimenting with this one...
not sure why I get errors in my repos whatever I decide to set this one to so far...

Reply:
can we not set it in the consuming repo? like i was [suggesting here](https://github.com/FoxSportsAustralia/stylelint-config-foxsports/pull/7#discussion_r745208003) – only the consuming repo knows what format the CSS will be in (based on its own build pipelines / stack choices)

Reply:
I have tried this and it works. So yes, we can remove it from consuming repos. [Here](https://bitbucket.foxsports.com.au/projects/MAR/repos/ionic/pull-requests/358/overview?commentId=164550) is a link to my experiment in ionic.